framework/bundles/org.jboss.tools.rsp.runtime.core/src/main/java/org/jboss/tools/rsp/runtime/core/model/installer/internal/DownloadRuntimeOperationUtilFactory.java

```diff
@@ -0,0 +1,52 @@
+package org.jboss.tools.rsp.runtime.core.model.installer.internal;
```

Comment:
Copyright please.

Reply:
I added copyright and removed some unused imports. I have no intention to rewrite some of the methods to use try-with-resources nor do i intend to change method signatures due to an unused variable at this time.